great_expectations/expectations/metrics/column_map_metrics/column_values_between.py

```diff
@@ -73,6 +75,30 @@ def _pandas(
         if min_value is not None and max_value is not None and min_value > max_value:
             raise ValueError("min_value cannot be greater than max_value")
 
+        # Use a vectorized approach for native numpy dtypes
+        if column.dtype in [int, float] and allow_cross_type_comparisons is not True:
```

Comment:
Should `allow_cross_type_comparisons` always be a boolean? I notice the default value is `None`. Would making the default value False and changing this to `allow_cross_type_comparisons is False` possible?

Reply:
That makes sense to me. I believe the reason the default is None is to be able to disambiguate between the case where it was not provided by the user at all (so isn't explicitly stored in the configuration).

That really makes me think `default` here was being used imprecisely to mix those cases. I think going back to fully disambiguate should be a different PR, but we can check for None and set to False here, and add a comment. LMK if you think that's a good solution.

Reply:
Updated.

Reply:
Sounds good to me.